src/components/molecules/NavBar/NavBar.scss

```diff
@@ -270,4 +272,30 @@ $border-radius: 28px;
   &__profile-popover {
     height: 100%;
   }
+
+  &__photobooth {
```

Comment:
My suggestion, to nip these misunerstandings in the bud

```scss
    &__photobooth-button {
    }

    &__photobooth-title {
    }
```

Reply:
fixed in 0f31efb